### DISCUSSION TITLE
Sort Issues Alphabetically

### DISCUSSION THREAD

**Discussion by dynamicdoyle** [community/community#43151] [Projects and Issues]
I title my issues like a hierarchy.
However, I can't sort alphabetically, so it's difficult keeping them organized.

Examples:
HEADER | NAV | Broken Anchor Link...
HOME | HERO | Webm Auto Loop Playback Freezes...
FOOTER | NAV | Missing Link...

**Comment by jfullstackdev** [comment]
these are the advanced features that might help 

https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests

I think the closest one is using Labels and sorting by date

**Reply by dynamicdoyle** [reply]
Searching in:title is helpful thanks.
Regarding your other comments: 
Unfortunately, I use labels as tags in a different manner. Trying to merge all of my issue naming hierarchies into tags would inundate or reduce the usability of labels for me.
I tried sorting by date, but unfortunately I don't create issues over time while following my hierarchies alphabetically, so it didn't really help.

**Comment by spenserblack** [comment]
If you organize your issues into a GitHub Project list view, you can sort by the issue title.

**Reply by dynamicdoyle** [reply]
Unfortunately I migrated my repo to another org and the projects didn't come with it, so I had to recreate and link the issues to those projects all over. 
If I had been able to sort by the issue title in the first place, then it would have saved me a lot of time recreating the projects with the remaining issues.

**Comment by dynamicdoyle** [comment]
Can this question be voted as a feature request? 
Or can I branch and pull this functionality to GitHub myself somewhere?

**Reply by queenofcorgis** [reply]
Hi there - with the `Product Feedback` label on this discussion, it will be categorized as a feature request 😄

**Comment by Elliria** [comment]
I would love this feature and will do what I can to spread the word so we can get some more up-votes on this.

**Comment by ivanperez-keera** [comment]
This feature is very much needed. I'm surprised that github always implements complex, "clever" sorting methods but almost never alphabetical sorting. It took years until org repos could be sorted alphabetically, and the order doesn't even stick.

If we sort by name, searching becomes easier (searching in a sorted list is log n and requires less attention that searching in a list that is not alphabetically sorted). Also, since we can customize the names at will, it gives us the ability to customize the order.

**Reply by Elliria** [reply]
I agree. A sorted list is a more natural environment for us to navigate in and, as you said, we can customize the sorting further by the names we use.

**Comment by torokati44** [comment]
Yes please!

**Comment by robinwilson16** [comment]
Please add this - sorting by title seems like a fairly basic feature and hopefully easy to implement

**Reply by robinwilsoncopc** [reply]
Replying over a year later to request again that it is added please

**Comment by rgleason** [comment]
I would like to sort Issues by title

is:issue state:closed sort:title 

It will help to organize work.

**Comment by Shane32** [comment]
ditto.  I'd like to sort issues by title also please.

**Comment by robinwilson16** [comment]
Yes please do this as it's been over 2.5 years since it was requested in this thread and I have been wanting to do it for about 10 years.

When you add sort options like sorting by reactions, or best match (?) but don't add the option to sort by title it does seem a little strange to omit what would seem to be the most obvious sort option whilst adding more obscure ones:
<img width="765" height="374" alt="image" src="https://github.com/user-attachments/assets/67c596d5-25ec-433f-8ab3-53468c806bc1" />

I suppose we could thumbs up the ones we want to appear at the top but if all thumb'd up once then un-thumb'd up but that still wouldn't define an exact order.

**Comment by mcmire** [comment]
Adding another request for this. Being able to sort issues alphabetically would be very valuable to be able to bulk-close duplicate issues.

I do realize that you can sort issues alphabetically in a Project but you can't bulk-close issues there for some reason (in general, not sure we why don't have the same capabilities in Projects as we in Issues, seems like an oversight).

**Comment by Danweel** [comment]
I'd also like to pitch in on this. I understand can probably sound odd to most - _who's sorting their bugs alphabetically??_

There's a few reasons this is helpful in the main area, and not just in our personal project spaces, where in we can opt to do that via a 'view'. It's not for _every_ repo out there, but for ones that have any sort of **naming convention** at all, it's helpful.

It probably doesn't make as much sense when you have >1000 (open) bugs or so, but **smaller projects would benefit** from having such a naming convention. Exactly what this convention comes down to really varies by project in my experience. It's good to have some flexibility here:

I do something like this, though this is just an example - the first could be location or type, with a possible second attribute to further organize the order.

GAMEPLAY: Level 1 - [bug title]
PAUSE: Settings - [bug title]

or

DOCS: contributing.rst - [topic to add]
REPO: conf.py - [some issue with the config]

Then I use **labels** for _issue severities_ (A, B, C, 1, 2, 3), _closure type_ (dupe, won'tfix), _hold type_ (more info needed, more testing needed), _regression concerns_ (reopened), and marking things that are '_requests_' to consider. These labels can accumulate as mutable and coloured information below the bug, keeping its title **immutable**:

Labels are a bit subjective, whereas it's not subjective or subject to change where the bug is located or what type it is. (Other projects might have other objective realities they want to reflect or enforce in their titles).

**Who it helps:** managers and people vetting the bugs other people input (usually to ensure that bugs are entered in a descriptive enough way, aren't already entered from some other perspective, don't refer to an already-closed issue, etc.), and smaller databases for visibility. Labels are nice but this affords a bit more control, optionally.

**Reply by robinwilson16** [reply]
Please add the feature before we all retire 🥲

**Comment by akifb** [comment]
If you are working in multi repo for same issue, then it would be helpful to see all related PRs grouped together. In our convention PR names start with issue number.
